README.md

```diff
@@ -38,13 +37,79 @@ helm install lfx-platform ./charts/lfx-platform -n lfx -f charts/lfx-platform/va
 
 ## Components
 
-The LFX v2 Platform includes the following core components:
-
-- **Traefik**: API Gateway and Ingress Controller
-- **OpenFGA**: Fine-Grained Authorization with Relationship-Based Access Control (ReBAC)
-- **Heimdall**: Identity and Access Proxy
-- **NATS**: Messaging and event streaming system
-- **OpenSearch**: Search and analytics engine
+LFX v2 includes the following infrastructure components:
+
+- **Traefik**: API Gateway and Ingress Controller.
+- **OpenFGA**: Fine-Grained Authorization with Relationship-Based Access
+  Control (ReBAC).
+- **Heimdall**: Access decision service, bridges Traefik to OpenFGA.
+- **NATS**: Messaging layer used by LFX v2 resource APIs to communicate with
```

Comment:
It's probably also worth calling out that we're using NATS k/v buckets for data storage.